zipkin-collector-service/src/main/scala/com/twitter/zipkin/collector/builder/CollectorServiceBuilder.scala

```diff
@@ -142,13 +129,6 @@ case class CollectorServiceBuilder[T](
       serverBuilder.adminHttpService map { _.addContext(path, new ConfigRequestHandler(adjustable)) }
     }
 
-    /* Start additional services (server sets) */
-    additionalServices foreach { builder =>
```

Comment:
we shouldn't take on random responsibilities such as registering arbitrary services.